data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,88 @@ mission "FW Epilogue: Danforth"
+			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there, I was lucky to get out myself"`
+			`	"No way you're from New Boston too! What are the chances?" you hastily respond.`
+			`	He chuckles, "Small galaxy I suppose." You both have a great time hearing of each others early lives but you wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and you head back to your ship.`
+				decline
+		event "plasma repeater advanced" 95
+		event "plasma repeater basics" 155
+	npc disable save
+		government "Test Dummy"
+		personality disables heroic staying
+		ship "Leviathan (Plasma Repeater)" `Hunk`
+	on complete
+		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon, increased fire rate even at lower damage per hit comes out with better overall results.`
+		conversation
+			scene "scene/plasma_scene"
+			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with a large gun much like an old Plasma Cannon, but somewhat different. Presumably, this is the weapon that you were just attacked with in orbit. "It runs alot hotter, you see," he says. "But the results are impressive."`
+			choice
+				`	"What was that?"`
+					goto what
+				`	"It looks impressive, but it didn't do too much damage."`
+					goto damage
+			label what
+			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That, boy, was the Plasma Repeater. Of course, it wasn't easy to find where opportunity lies when everyone could have taken a crack at it."`
+			`	"When will it be available for sale?" you ask.`
+				goto available
+			label damage
+			`	"Well, you are famous for a reason I suppose." he says. "The idea is that by shrinking the concentrations of superheated plasma we can offload many more bursts onto a target, the effects on said target result in faster build up of heat and of course damage. Of course, big ships can ward of these effects much longer than a smaller ship can, it's always been a notable issue with the orginal design, but it is a hell of a lot more impressive."`
+			`	"That is an interesting idea," you say. "Are they available for sale yet?"`
+			label available
+			`	"Well, I should have an initial model for sale in a month or so," he says. "I'll be interested to see what uses you or anyone else might find for them. Anyway, thanks for your help."`
+			`	"How about we celebrate with some drinks? I know a quaint little bar not far from here."
+			choice
+				`	"I have time, lets celebrate!"`
+				`	"I'll pass on that but it was good checking in on you."`
+					decline
```

Comment:
Does this need to be a `goto end` or something? This isn't in the `on offer` section, this is `on complete`. Could be me just not knowing the syntax, though.

Reply:
you need to use one of the conversation enders (accept, decline, defer) if you want to end a conversation without it reaching the end